pkgs/applications/video/mpv/scripts/mpv-discord.nix

```
@@ -0,0 +1,45 @@
+{
+  fetchFromGitHub,
+  buildGoModule,
+  buildLua,
+  lib,
+}:
+let
+  version = "1.6.1";
+
+  src = fetchFromGitHub {
+    owner = "tnychn";
+    repo = "mpv-discord";
+    rev = "v${version}";
+    hash = "sha256-Uvm4McZR88s9tsstqLz7O2F3KkqsGlGrDoL8RjgQnVI=";
```

```suggestion
    hash = "sha256-P1UaXGboOiqrXapfLzJI6IT3esNtflkQkcNXt4Umukc=";
```

Remember to check CI failures.